### DISCUSSION TITLE
Is there a nice grafana integration guide somewhere?

### DISCUSSION THREAD

**Discussion by mdlam92** [geldata/gel#4696] [Q&A]
Howdy! I'm trying to use grafana to pull and display data from my edgedb instance, I'm using an unmaintained graphql plugin for grafana and not having the best of time. Is there a plugin or a guide for integrating with grafana yet?

**Comment by Sikarii** [comment]
@washed shared their work in [our Discord server](https://discord.gg/edgedb), the repository is at [washed/edgedb-grafana-backend](https://github.com/washed/edgedb-grafana-backend).

Is this close to what you're looking for?

**Reply by mdlam92** [reply]
this might be what I'm looking for, I'll give a whirl, thanks!

**Reply by Sikarii** [reply]
Additionally there's [edgedb/edgedb-grafana-frontend](https://github.com/edgedb/edgedb-grafana-frontend) that fetches from the EdgeDB HTTP endpoint.
